lib/ex_rabbit_m_q/consumer.ex

```diff
@@ -553,6 +566,27 @@ defmodule ExRabbitMQ.Consumer do
         end
       end
 
+      def xrmq_basic_nack(delivery_tag, opts \\ [], state) do
```

Comment:
We should be able to handle multiple delivery tags here since `nack` allows it.

Could we, please, add some function heads to accommodate this case?

Reply:
Also, isn't delivery tag an integer(), not a String.t()?